EIPS/eip-future_salaries.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 3267
+title: Giving Ethereum fees to Future Salaries
+author: Victor Porton (@vporton), Victor Porton <porton@narod.ru>
+discussions-to: https://ethereum-magicians.org/t/discussion-of-eip-3267/5343
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-02-13
+---
+
+## Simple Summary
+Transfer a part of Ethereum transfer/mining fees to Future Salaries contract
+
+## Abstract
+Transfer a part (exact fractions - TBD) of mining/transfer fees to the `DonateETH` contract configured to transfer to `SalaryWithDAO` contract.
+
+## Motivation
+This proposal solves two problems at once:
+
+1. It provides a big amount of "money" to common good producers. That obviously personally benefits common good producers, allowing them to live better human lives, it increases peoples' and organizations' both abilities and incentives to produce common goods. That benefits the humanity as a whole and the Ethereum ecosystem in particular. See more in the discussion why it's crucial.
+
+2. This would effectively decrease circulating ETH supply. The necessity to decrease the (circulating) ETH supply (by locking ETH in Future Salaries system for a long time) is a well-known important thing to be done.
+
+Paradoxically, it will directly benefit miners/validators, see the discussion.
+
+## Specification
+(TBD)
+
+`SalaryWithDAO` = `TBD`
+
+`DefaultDAOInterface` = `TBD`
+
+Prior to `FORK_BLOCK_NUMBER`, [SalaryWithDAO](https://github.com/vporton/future-contracts/blob/master/contracts/SalaryWithDAO.sol) and [DefaultDAOInterface](https://github.com/vporton/future-contracts/blob/master/contracts/DefaultDAOInterface.sol) contracts will be deployed to the network and exist at the above specified addresses.
```

Comment:
This code should be either moved inline or added to `../assets/eip-3267/...`.  If it is a core part of the specification, inline is preferred.

Reply:
Inline would be hard to read because it is multiple files and some are long. I will put into `../assets/eip-3267/`